VERSION.txt

```diff
@@ -1 +1 @@
-0.6.0
+0.6.1
```

Comment:
[These have meanings](https://semver.org/)! Since you have added new backwards compatible functionality this is a minor version increment. The new version should be `0.7.0`.

Reply:
Changed version number